diff --git a/./temp/Stories/intro.exo b/./temp/ModStories/intro.exo
index b1c36f7..4b47f20 100644
--- a/./temp/Stories/intro.exo
+++ b/./temp/ModStories/intro.exo
@@ -725,8 +725,16 @@
 		
 		Your earliest memories are of your fourth birthday. There's a big cake with a neat holo effect that _looks_ like fire, but when you reach out to touch it all you get is a handful of icing. Yummy!
 		
-		Your favorite present is:
-		
+		// modded content
+		-
+		~set card = whiteFlower
+		Your mom also gives you a beautiful flower as a gift. It was dried and pressed flat so it could be preserved during the long journey.
+		You haven't seen flowers outside of picture books, and it's really pretty!
+		Later, when you're older, you'll look at the flower and think about how carefully and gently your mom must have folded down each soft petal.
+
+		-		
+		Your other favorite present is:
+
 		** Your mom's gardening trowel
 			~set mem_work_farm + 5
 			~set skill_biology + 5
